web/templates/user_group_settings/user_group_settings.hbs

```diff
@@ -43,7 +43,7 @@
                     {{> ../settings/settings_save_discard_widget section_name="group-permissions" }}
                 </div>
 
-                {{> group_permissions prefix="id_" group_setting_labels=group_setting_labels}}
+                {{> group_permissions prefix="id_" group_setting_labels=group_setting_labels.group}}
```

Comment:
I think this suggests we have something with a bad name. Maybe `group_setting_labels=all_group_setting_labels.group` would address it?